simfire/utils/config.py

```diff
@@ -794,7 +813,7 @@ def _load_fire(self, pos: Optional[Tuple[int, int]] = None) -> FireConfig:
             return FireConfig((pos_x, pos_y), diagonal_spread, max_fire_duration, seed)
         elif fire_init_pos_type == "historical":
             return FireConfig(
-                (self.historical_layer.fire_start_y, self.historical_layer.fire_start_x),
+                (self.historical_layer.fire_start_x, self.historical_layer.fire_start_y),
```

Comment:
I'm nervous about this switch. Did you verify this?

Reply:
Yes, I verified it. I see this as setting the value of `fire_initial_position`:

https://github.com/mitrefireline/simfire/blob/5d76a16de45e7058e5080f2cf4c5b5b2f9f1d0ae/simfire/utils/config.py#L186

which is then leveraged to set the fire start position for a new Simulation here:

https://github.com/mitrefireline/simfire/blob/5d76a16de45e7058e5080f2cf4c5b5b2f9f1d0ae/simfire/sim/simulation.py#L565

Let me know if you think the switch is incorrect!